website/src/blogs/install-wordpress-using-dynamic-nfs-provisioner.md

```diff
@@ -0,0 +1,217 @@
+
+### Prerequisites for deploying dynamic NFS provisioner
+- Minimum Kuberentes Version 1.18
+- NFS Client must be installed on all nodes where the application pod schedules.
+  For Ubuntu/Debian run below command to install nfs client:
+  ```sh
+  sudo apt install nfs-common
+  ```
+- Storage provider(supports dynamic provisioning) to serve backend volume claim requests. In this blog, we are going to use OpenEBS localpv-hostpath as a backend storage provider.
+
+### Install NFS Provisioner
+  NFS provisioner will dynamically provision NFS volumes, install NFS provisioner using below helm commands and it will deploy NFS provisioner with basic configurations, NFS provisioner also accepts [values.yaml](https://github.com/openebs/dynamic-nfs-provisioner/tree/develop/deploy/helm/charts#configuration)
+
+  - To install NFS provisioner add repository and then install nfs-provisioner
+
+    #### Add OpenEBS repo for installation
+    - Use the below command to add helm repo for OpenEBS
+
+          $ helm repo add openebs https://openebs.github.io/charts
+    - Once repo has been added successfully, update helm repo using the following command:
+
+          $ helm repo update
+
+    #### Installing OpenEBS NFS Provisioner
+      Once the helm repo addition is successful, run the `helm install` command as specified below, in this command, we are disabling the local-pv provisioner which doesn’t require for NFS Provisioner to work and configure __.nfsStorageClass.backendStorageClass__ corresponding to backend StorageProvider
+
+      ```sh
+      helm install openebs openebs/openebs -n openebs --create-namespace \
+        --set legacy.enabled=false \
+        --set ndm.enabled=false \
+        --set ndmOperator.enabled=false \
+        --set localProvisioner.enabled=false \
+        --set nfs-provisioner.enabled=true \
+        --set nfs-provisioner.nfsStorageClass.backendStorageClass=<backend_sc_name>
+      ```
+      - After successful helm installation, you can view the __openebs-nfs-provisioner__ and __openebs-kernel-nfs__ StorageClass will get deployed.
+
+        **Output**:
+        ```sh
+        [develop@develop ~]$ kubectl get po -n openebs
+        NAME                                           READY   STATUS    RESTARTS   AGE
+        openebs-nfs-provisioner-79b6ccd59-v8p6s        1/1     Running   0          4m12s
+
+        [develop@develop~]$ kubectl get sc
+        NAME                 PROVISIONER         RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
+        openebs-kernel-nfs   openebs.io/nfsrwx   Delete          Immediate              false                  67s
+        ```
+
+     **Optional(installation of local-pv-provisioner)**
+        The below command will install OpenEBS local-pv provisioner along with NFS  provisioner, as stated above local-pv is used as backend StorageProvider in this blog
+
+        ```sh
```

Comment:
For this block more formatting is needed I believe. The backticks (```) are being visible and the Note section is inside code block.

Reply:
Updated the text with required indentation